civictechprojects/views.py

```diff
@@ -22,7 +22,7 @@
 
 #TODO: Set getCounts to default to false if it's not passed? Or some hardening against malformed API requests
 
-@cache_page(300) #cache duration in seconds, cache_page docs: https://docs.djangoproject.com/en/2.1/topics/cache/#the-per-view-cache
+@cache_page(1200) #cache duration in seconds, cache_page docs: https://docs.djangoproject.com/en/2.1/topics/cache/#the-per-view-cache
```

Comment:
When we get a task scheduler of some type this should be set to the scheduler interval, I just wanted a longer value for more testing.